providers/implementations/include/prov/ciphercommon.h

```diff
@@ -362,6 +362,30 @@ const OSSL_PARAM * name##_settable_ctx_params(ossl_unused void *cctx,          \
     return name##_known_settable_ctx_params;                                   \
 }
 
+struct ossl_cipher_get_ctx_param_list_st {
```

Comment:
This seems a bit unfortunate here..

Reply:
Unfortunate in what sense?

Reply:
This is a manually created structure as opposed to everything else - could it be generated and this would then just be an extern?

Reply:
I agree with @paulidale